packages/jira-adapter/src/utils.ts

```diff
@@ -59,6 +59,16 @@ export const findObject = (elements: Element[], name: string): ObjectType | unde
   return type
 }
 
+export const findObjects = (elements: Element[], names: string[]): ObjectType[] | undefined => {
+  const types = elements.filter(isObjectType).filter(element => names.includes(element.elemID.name))
+
+  if (types.length !== names.length) {
```

Comment:
We can be more explicit here and add 
```suggestion
if (types.length < names.length) {
```

Reply:
As the types cannot be bigger than names (object type names are unique) I do not think it makes a difference, and IMO it is more readable with `!==`